backend/mobilityAI/Controllers/SensorDataController.cs

```diff
@@ -160,9 +170,31 @@ public IActionResult AddData(IFormFile AccelerometerFile, IFormFile GyroscopeFil
             _context.GyroscopeData.AddRange(GyroscopeObjects);
             _context.SaveChanges();
 
+            
+            var accelMs = new MemoryStream();
+            AccelerometerFile.OpenReadStream().CopyTo(accelMs);
+
+            var gyroMs = new MemoryStream();
+            GyroscopeFile.OpenReadStream().CopyTo(gyroMs);
+
+            var callbackId = Guid.NewGuid().ToString();
+
+            MultipartFormDataContent form = new MultipartFormDataContent();
+
+            form.Add(new StringContent(SERVER_URL + "api/SensorData/MlCallback?Id=" + callbackId), "callback_url");
+            //form.Add(new StringContent(callbackId), "callback_id");
+            form.Add(new ByteArrayContent(accelMs.ToArray()), "file[]", AccelerometerFile.FileName);
```

Comment:
Why not use the results from the readData() function since it already returns a list?

Reply:
If you're talking about the `accelMs.ToArray()` it returns an array of bytes, `readData()` is a list split by newlines.

Reply:
Can probably combine those later so that we use MemoryStream.ToArray() for the db stuff as well.